lib/oli/delivery/sections/blueprint.ex

```diff
@@ -159,12 +159,13 @@ defmodule Oli.Delivery.Sections.Blueprint do
             "start_date" => nil,
             "end_date" => nil,
             "title" => title,
-            "requires_payment" => false,
-            "payment_options" => "direct_and_deferred",
-            "pay_by_institution" => false,
-            "registration_open" => false,
-            "grace_period_days" => 1,
-            "amount" => Money.new(:USD, "25.00"),
+            "requires_payment" => attrs["requires_payment"] || false,
+            "payment_options" => attrs["payment_options"] || "direct_and_deferred",
+            "pay_by_institution" => attrs[""] || false,
+            "registration_open" => attrs["registration_open"] || false,
+            "grace_period_days" => attrs["grace_period_days"] || 1,
+            "amount" =>
+              Money.new(attrs["amount"]["currency"] || :USD, attrs["amount"]["amount"] || "25.00"),
```

Comment:
Should we only assign the default currency and amount values if both are missing? I am imagining the improbable case where the original value is EUR 30, and we end up having USD 30 if the currency value is missing.

Reply:
There were already default values for the amount, so I just set the context for handling those values during the import. I couldn't find where the currency is defined. @eliknebel, do we use any currency other than 'USD'?

Reply:
Yes I agree with Nico, we should only assign the default currency if both amount and currency are missing to avoid unintentionally changing the underlying value